bundles/org.openhab.binding.velbus/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -653,4 +969,401 @@
+			<channel id="CH6" typeId="ledFeedback"/>
+		</channels>
+	</channel-group-type>
+	<channel-group-type id="8channelFeedbackModule">
+		<label>Feedback</label>
+		<description>
+			This is a generic module with 8 feedback channels.
+		</description>
+		<channels>
+			<channel id="CH1" typeId="ledFeedback"/>
+			<channel id="CH2" typeId="ledFeedback"/>
+			<channel id="CH3" typeId="ledFeedback"/>
+			<channel id="CH4" typeId="ledFeedback"/>
+			<channel id="CH5" typeId="ledFeedback"/>
+			<channel id="CH6" typeId="ledFeedback"/>
+			<channel id="CH7" typeId="ledFeedback"/>
+			<channel id="CH8" typeId="ledFeedback"/>
+		</channels>
+	</channel-group-type>
+	<channel-group-type id="32channelFeedbackModule">
+		<label>Feedback</label>
+		<description>
+			This is a generic module with 32 feedback channels.
+		</description>
+		<channels>
+			<channel id="CH1" typeId="ledFeedback"/>
+			<channel id="CH2" typeId="ledFeedback"/>
+			<channel id="CH3" typeId="ledFeedback"/>
+			<channel id="CH4" typeId="ledFeedback"/>
+			<channel id="CH5" typeId="ledFeedback"/>
+			<channel id="CH6" typeId="ledFeedback"/>
+			<channel id="CH7" typeId="ledFeedback"/>
+			<channel id="CH8" typeId="ledFeedback"/>
+			<channel id="CH9" typeId="ledFeedback"/>
+			<channel id="CH10" typeId="ledFeedback"/>
+			<channel id="CH11" typeId="ledFeedback"/>
+			<channel id="CH12" typeId="ledFeedback"/>
+			<channel id="CH13" typeId="ledFeedback"/>
+			<channel id="CH14" typeId="ledFeedback"/>
+			<channel id="CH15" typeId="ledFeedback"/>
+			<channel id="CH16" typeId="ledFeedback"/>
+			<channel id="CH17" typeId="ledFeedback"/>
+			<channel id="CH18" typeId="ledFeedback"/>
+			<channel id="CH19" typeId="ledFeedback"/>
+			<channel id="CH20" typeId="ledFeedback"/>
+			<channel id="CH21" typeId="ledFeedback"/>
+			<channel id="CH22" typeId="ledFeedback"/>
+			<channel id="CH23" typeId="ledFeedback"/>
+			<channel id="CH24" typeId="ledFeedback"/>
+			<channel id="CH25" typeId="ledFeedback"/>
+			<channel id="CH26" typeId="ledFeedback"/>
+			<channel id="CH27" typeId="ledFeedback"/>
+			<channel id="CH28" typeId="ledFeedback"/>
+			<channel id="CH29" typeId="ledFeedback"/>
+			<channel id="CH30" typeId="ledFeedback"/>
+			<channel id="CH31" typeId="ledFeedback"/>
+			<channel id="CH32" typeId="ledFeedback"/>
+		</channels>
+	</channel-group-type>
+	<channel-group-type id="oledDisplay">
+		<label>O-LED Display</label>
+		<channels>
+			<channel id="MEMO" typeId="text">
+				<label>Memo</label>
+			</channel>
+			<channel id="SCREENSAVER" typeId="switch">
+				<label>Screensaver</label>
+			</channel>
+		</channels>
+	</channel-group-type>
+	<channel-group-type id="thermostat">
+		<label>Thermostat</label>
+		<description>This is a thermostat that supports heating/cooling and comfort/day/night/safe modes.</description>
+		<category>Temperatures</category>
+		<channels>
+			<channel id="CURRENTTEMPERATURESETPOINT" typeId="temperatureSetpoint">
```

Comment:
Since all new or updated channels can you make them camel case. There is no required naming convention for channels, but we strongly suggest to use camel case or a variant, but no uppercase channels. 
```suggestion
			<channel id="currentTemperatureSetpoint" typeId="temperatureSetpoint">
```

Reply:
Modified in commit 780323c